dev/TreeView/TreeViewItem.cpp

```diff
@@ -321,20 +321,14 @@ void TreeViewItem::OnPropertyChanged(const winrt::DependencyPropertyChangedEvent
         {
             winrt::IInspectable value = args.NewValue();
 
-            // ItemsSource change happens during measuring.
```

Comment:
Do you know what changed so we do not need this?

Reply:
We use to handle TreeView::ItemsSource and TreeViewItem::ItemsSource separately, I'm guessing that may trigger different layout passes and creates conflicts.

Now TreeViewItem::ItemsSource just sets ItemsSource property on root node and let TreeViewItem do the actual work.